runtime/lib/common/system_desc.cpp

```diff
@@ -93,6 +93,48 @@ getAllDeviceConnections(const vector<::tt::tt_metal::Device *> &devices) {
   return allConnections;
 }
 
+// Gather all physical cores by type for the device using metal device APIs
+static flatbuffers::Offset<::tt::target::ChipPhysicalCores>
+createChipPhysicalCores(const ::tt::tt_metal::Device *device,
+                        flatbuffers::FlatBufferBuilder &fbb) {
+
+  std::vector<::tt::target::Dim2d> worker_cores, dram_cores, eth_cores,
+      eth_inactive_cores;
+
+  auto logical_grid_size = device->logical_grid_size();
```

Comment:
Yeah? I was wondering if we liked auto.  The 80 character limit was such a bother, I was doing everything I could everywhere to avoid line-braks for the simplest of code, but can change this to CoreCoord, yep.

Reply:
I personally avoid using it, much easier later on to figure out what is code doing. I only use it for iterators and really long types which need namespace reference.

Reply:
sounds good, updated, resolved.